lib/msal-core/chai.conf.js

```diff
@@ -0,0 +1,5 @@
+let chai = require('chai');
+
+chai.config.includeStack = false
```

Comment:
I believe you can set this once if in the write place and still import chai from the lib. No need to export here.
I think it is a global setting.
The main question is where do you set it so it runs before the test

Reply:
you could add to files in karma conf